docs/src/administration/users.md

```diff
@@ -3,136 +3,182 @@ title: "User administration"
+To change the user permissions, follow these steps:
 
-Please make sure you have the correct access rights and the repository exists.
-```
+{{< codetabs >}}
+---
+title=In the console
+file=none
+highlight=false
+---
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
 
-despite their **Project Admin** access to the project.
+<--->
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+`platform user:role`
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+  ```bash
+  platform user:role user1@example.com -r contributor
+  ```
 
-```bash
-$ git clone <project>@git.<region>.platform.sh:<project>.git
-```
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command `platform redeploy`.
+
+{{< /codetabs >}}
+
+
+### Transfer project ownership
+
+You can transfer your plan ownership to a different [organization](/administration/organizations.md) anytime.
+You have to be the organization owner or an organization user with the **manage plans** permission.
+
+1. Invite the new organization owner as a **Project Admin**.
+2. Submit a [support ticket](https://console.platform.sh/-/users/~/tickets) from your organization owner's account to ask for the transfer.
+
+Once the transfer is completed, the new organization can administer all project settings, billing, and gets the subscription charges.
 
-## Transfer ownership
 
-If you're the owner of a project, and want to transfer it to a different user, first invite that user as a project administrator and then submit a support ticket from your owner account to ask for the transfer.
+## Troubleshooting
 
-Transfering a project automatically transfers the upcoming subscription charges to the new owner.
+If you have setup an external integration to GitHub, GitLab, or Bitbucket, and your users can't clone the project locally, select [Source Integrations](integrations/source/troubleshooting.md).
```

Comment:
```suggestion
If you have setup an external integration to GitHub, GitLab, or Bitbucket, and your users can't clone the project locally, see how to [troubleshoot source integrations](integrations/source/troubleshooting.md).
```

Although the first clause might assume the user knows enough to fix the problem (like we were discussing with the FAQ). But it might be OK.